charts/logging-operator/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+type: application
+name: logging-operator
+version: 0.0.0
+appVersion: latest
+kubeVersion: ">=1.22.0-0"
+description: Logging operator for Kubernetes based on Fluentd and Fluentbit.
+keywords:
+  - logging
+  - fluentd
+  - fluentbit
+home: https://kube-logging.github.io
+sources:
+  - https://github.com/kube-logging/logging-operator
+  - https://github.com/kube-logging/helm-charts/tree/main/charts/logging-operator
+annotations:
+  artifacthub.io/changes: |
+    - kind: changed
+      description: Moved to to location
+  artifacthub.io/images: |
+    - name: logging-operator
+      image: ghcr.io/kube-logging/logging-operator:4.2.1
```

Comment:
I beleive we should leave it off to avoid getting out of sync with the appVersion all the time.

> By default, Artifact Hub will try to extract the containers images used by Helm charts from the manifests generated from a dry-run install using the default values. If you prefer, you can also provide a list of containers images manually by using this annotation. Providing the information manually also allows adding some extra information, like the platforms supported by each of the images.

```suggestion
```